libraries/lib-audio-devices/AudioIOBase.h

```diff
@@ -233,7 +233,7 @@ class AUDIO_DEVICES_API AudioIOBase /* not final */
     * methods use whatever device is specified by the preferences.  If a
     * stream *is* running, naturally they manipulate the mixer associated
     * with that stream.  If no mixer is available, output is emulated and
-    * input is stuck at 1.0f (a gain is applied to output samples).
+    * input is stuck at 1.0f (a volume gain is applied to output samples).
```

Comment:
is `a volume gain` is intentional here?

Reply:
Yeah, kind of. Could have been "(volume is applied", but that's also acceptable I think.